src/OpenLoco/src/GameCommands/Terraform/LowerLand.h

```diff
@@ -33,5 +35,6 @@ namespace OpenLoco::GameCommands
         }
     };
 
+    uint32_t lowerLand(const LowerLandArgs& args, std::set<World::Pos3, World::LessThanPos3>& removedBuildings, const uint8_t flags);
     void lowerLand(registers& regs);
```

Comment:
at what point do we remove this one?

Reply:
When the game command runner no longer relies on registers, but can un(serialize) our game command structs instead.